src/lib/legacy.js

```diff
@@ -697,6 +697,39 @@ export function getBuyGlpFromAmount(toAmount, fromTokenAddress, infoTokens, glpP
   return { amount: fromAmount, feeBasisPoints };
 }
 
+export function getLowestFeeTokenForBuyGlp(
```

Comment:
i guess it's better to avoid adding anything new to legacy.js?

Reply:
Done!